### PR TITLE
Appng 143 - Copy missing examples from showcase to demo app

### PR DESCRIPTION
**Pull request by jramcast**
Missing demostrations from appverse-web-html5 have been copied to the demo app in order to use the latter as the new showcase application.

Changes were required in terms of design and navigation to make usability easier in desktop and mobile devices.

### CODE REVIEW

**Comment by jramcast** [comment]
checking conflicts...

File: config/test/protractor.e2e.dist.conf.js [hunk 1]
```diff
@@ -8,6 +8,17 @@ settings.specs = [
     '../../test/e2e/**/*.js',
 ];
 
+// Enable PhantomJS testing here.
+// Its seems that there are no random failures when testing with
+// non instrumented code
+// TODO: remove Phantomjs and use real browsers with Vagrant
+settings.capabilities = {
+    browserName: 'phantomjs',
+    //'phantomjs.binary.path': 'node_modules/.bin/phantomjs' + (process.platform === 'win32' ? '.cmd' : ''),
```
Comment (aecz) [inline config/test/protractor.e2e.dist.conf.js]: We need those tests to work in Windows and Unix environment.

Comment (jramcast) [inline config/test/protractor.e2e.dist.conf.js]: It works. Just replaced the previous line for ```require('phantomjs').path ```, which resolves the binary path in both environments and seems more elegant

Comment (aecz) [inline config/test/protractor.e2e.dist.conf.js]: Ok, perfect. Please remove commented code then.

**Comment by jramcast** [comment]
Done!

File: config/test/protractor.e2e.dist.conf.js (hunk 1 above)
Comment (jramcast) [inline config/test/protractor.e2e.dist.conf.js]: Done!

**Pull request merged**